packages/playground/src/explorer/views/farms.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <view-layout>
+    <v-row>
+      <v-col>
+        <filters
+          v-model="filterFarmInputs"
+          @update:model-value="inputFiltersReset"
+          :form-disabled="isFormLoading"
+          v-model:valid="isValidForm"
+        />
+        <FarmsTable :items="farms" :loading="loading" :selectedFarm="selectedFarm" @open-dialog="openDialog" />
+      </v-col>
+    </v-row>
+    <farmDialog v-if="selectedFarm" :openDialog="isDialogOpened" :farm="selectedFarm" @close-dialog="closeDialog" />
```

Comment:
close dialogue should be triggered with any event that makes the dialogue close
```suggestion
    <farmDialog v-if="selectedFarm" :openDialog="isDialogOpened" :farm="selectedFarm" @close-dialog="closeDialog"  @update:model-value="closeDialog" />
```